plugins/org.eclipse.n4js.external.libraries/shipped-code/runtime/n4js-runtime-fetch/src/n4js/Request.n4jsd

```diff
@@ -10,30 +10,38 @@
  */
 @@Global @@ProvidedByRuntime
 
-
 import {Body} from "fetch/Body";
 import {Method} from "fetch/Method";
+import {ReferrerPolicy} from "fetch/ReferrerPolicy";
```

Comment:
What's this change about?

Reply:
Sry this change has nothing to do with this PR. But our shipped code is out-of-date so I update it here.